### PR TITLE
Added Teku docker instructions

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## Pull Request Description
Add instructions for running Docker image

### CODE REVIEW

File: docs/HowTo/Get-Started/Run-Docker-Image.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+              "--data-path=/opt/besu/data/data",
+              "--host-whitelist=*",
+              "--rpc-http-enabled",
+              "--rpc-http-cors-origins=*",
+              "--rpc-http-api=ETH,NET,CLIQUE,DEBUG,MINER,NET,PERM,ADMIN,EEA,TXPOOL,PRIV,WEB3",
+              "--miner-enabled=true",
+              "--miner-coinbase=0xfe3b557e8fb62b89f4916b721be55ceb828dbd73",
+              "--min-gas-price=0"]
+    volumes:
+      - ./:/opt/besu/data
```
Comment (EdJoJob) [inline docs/HowTo/Get-Started/Run-Docker-Image.md:83]:
```suggestion
      - ./besu:/opt/besu/data
```

File: docs/HowTo/Get-Started/Run-Docker-Image.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  teku_node:
+    image: pegasyseng/teku:develop
+    command: ["--eth1-deposit-contract-address=dddddddddddddddddddddddddddddddddddddddd",
+              "--eth1-endpoint=http://besu_node:8545",
+              "--validators-key-file=/opt/teku/data/validator_keys",
+              "--p2p-port=9000",
+              "--rest-api-enabled=true",
+              "--rest-api-docs-enabled=true"]
+    volumes:
+      - ./:/opt/teku/data
```
Comment (EdJoJob) [inline docs/HowTo/Get-Started/Run-Docker-Image.md:97]:
```suggestion
      - ./teku:/opt/teku/data
```

File: docs/HowTo/Get-Started/Run-Docker-Image.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+    ```bash
+    docker run -p 9000:9000 -p 5051:5051 -e TEKU_REST_API_ENABLED=true -e TEKU_P2P_PORT=9000 -e TEKU_LOG_FILE=/var/lib/teku/LOG --mount type=bind,source=/Users/user1/teku/,target=/var/lib/teku pegasyseng/teku:develop --eth1-deposit-contract-address=dddddddddddddddddddddddddddddddddddddddd --eth1-endpoint=http://102.10.10.1:8545 --validators-key-file=var/lib/teku/validator_keys.yaml
+    ```
+
+
+## Exposing ports
+
+Expose ports for P2P peer discovery, metrics, and REST APIs. You need
+to expose the ports to use the default ports or the ports specified using
```
Comment (MadelineMurray) [inline docs/HowTo/Get-Started/Run-Docker-Image.md:37]: Expose the default ports or the ports specified using: 

Suggested edit to make active

File: docs/HowTo/Get-Started/Run-Docker-Image.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+              "--rest-api-docs-enabled=true"]
+    volumes:
+      - ./:/opt/teku/data
+    ports:
+      - "9000:9000"
+      - "5051:5051"
+```
+
+Run `docker-compose up` in the directory containing the `docker-compose.yml` file
+to start the container.
```
Comment (MadelineMurray) [inline docs/HowTo/Get-Started/Run-Docker-Image.md:104]: To start the container, run `docker-compose up` .... 

Suggest moving the outcome to the start of the sentence.

File: docs/HowTo/Get-Started/Run-Docker-Image.md (hunk 3 above)
Comment (bgravenorst) [reply docs/HowTo/Get-Started/Run-Docker-Image.md:37]: Done

File: docs/HowTo/Get-Started/Run-Docker-Image.md (hunk 4 above)
Comment (bgravenorst) [reply docs/HowTo/Get-Started/Run-Docker-Image.md:104]: Done

Comment (bgravenorst) [reply docs/HowTo/Get-Started/Run-Docker-Image.md]: Done

**Pull request merged**